src/Symfony/Component/Console/Helper/FormatterHelper.php

```diff
@@ -73,6 +73,26 @@ public function formatBlock($messages, $style, $large = false)
     }
 
     /**
+     * Truncates a message to the given length.
+     *
+     * @param string $message
+     * @param int    $length
+     * @param string $suffix
+     *
+     * @return string
+     */
+    public function truncate($message, $length, $suffix = '...')
+    {
+        $computedLength = $length - strlen($suffix);
+
+        if ($computedLength > strlen($message)) {
```

Comment:
Actually, there's the [`Helper::strlen()`](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Console/Helper/Helper.php#L52), so it would be better to call `$this->strlen()`.